scripts/extensions/sams/package.json

```diff
@@ -7,7 +7,7 @@
     "watch": "tsc -watch -p src"
   },
   "dependencies": {
-    "@superdesk/common": "^0.0.13",
+    "@sourcefabric/common": "0.0.33",
```

Comment:
@tomaskikutis Where is the code for v0.0.33, I don't see it in https://github.com/superdesk/sourcefabric-common

Reply:
my bad forgot to push it - it's there now